http-api/src/error.rs

```diff
@@ -87,6 +87,10 @@ pub enum Error {
     /// An error occurred with env variables.
     #[error(transparent)]
     VarError(#[from] std::env::VarError),
+
+    /// An error occurred during the identity resolving.
+    #[error(transparent)]
+    IdentityResolveError(#[from] radicle_common::cobs::ResolveError),
```

Comment:
We usually avoid repeating `Error` in the variants, so I'd rename this to `IdentityResolve`. You could also fix the one above to just `Var` or `Env` maybe.

Reply:
Ok great, I also fixed the `SiweVerificationError` and the `SiweParseError`